README.md

```diff
@@ -155,9 +124,9 @@ The ownership voucher created for the device in the previous step needs to be im
 
 ### <a name="boot-device"></a>Boot the Device to Have it Configured
 
-When an SDO-enabled device boots, it starts the SDO process which contacts the SDO owner services to configure the device for this Horizon instance. **Back on your VM device**, simulate the booting of the device and watch SDO configure it:
+When an SDO-enabled device boots, it starts the SDO process which first contacts the rendezvous server, which redirects it to the SDO owner services in your Horizon instance. **Back on your VM device**, simulate the booting of the device and watch SDO configure it:
```

Comment:
I think breaking the first sentence into two helps with understanding the flow.

When an SDO-enabled device boots, it starts the SDO process which first contacts the rendezvous server. This rendezvous server then redirects it to the SDO owner services in your Horizon instance. **Back on your VM device**, simulate the booting of the device, and watch SDO configure it:

Reply:
good suggestion